lib/matplotlib/colorbar.py

```diff
@@ -509,6 +510,66 @@ def __init__(self, ax, mappable=None, *, cmap=None,
         # Set the cla function to the cbar's method to override it
         self.ax.cla = self._cbar_cla
 
+    @property
+    def locator(self):
+        """
+        Major `~.Locator` being used for colorbar.
+        """
+        return self._long_axis().get_major_locator()
+
+    @locator.setter
+    def locator(self, loc):
+        """
+        Set the major `~.Locator` being used for colorbar.
+        """
```

Comment:
I'd leave out the docstring of the setters. It's not rendered in HTML and cannot easily be accessed from interactive sessions. And it's always trivially "set the thing already described in the getter".